### ISSUE TITLE
No table of contents in offline epub version

### ISSUE THREAD

**Issue by kevzettler** [godotengine/godot-docs#9279]
**Your Godot version:**
4.2
**Issue description:**
The epub version of Godot documentation doesn't seem to contain a table of contents. I've tried reading in AppleBooks, Calibre Ebook-viewer, and converting to MOBI for Amazon Kindle. This makes the epub documentation like impossible to use. Further more it seems to significantly slow down Apple books and often crash Apple books. Not sure if that is from size of content or malformed content.

**URL to the documentation page (if already existing):**
https://github.com/godotengine/godot-docs/blob/master/README.md#download-for-offline-use

**Comment by Calinou** [comment]
> Further more it seems to significantly slow down Apple books and often crash Apple books. Not sure if that is from size of content or malformed content.

Unfortunately, there's no way to split the ePub into smaller files while keeping internal links working.

**Comment by skyace65** [comment]
@Calinou that PR did fix the issue. I just double checked and I'm able to open the table of contents on the latest EPUB build in the latest version of Calibre on my Windows 10 and 11 machine. Here's a screenshot of it on an XR page that was updated a week ago in #9113.
![EPUBTable](https://github.com/godotengine/godot-docs/assets/6080368/bbaefc92-f8b6-436c-9b5d-d9a6f79c7207)

Apple Books specific issues are already known, see #9062. No idea if that issue is our end or their end though.

**Comment by skyace65** [comment]
@kevzettler Could you please double check that it's not available in Calibre, and that you're on the latest version of Calibre. Also what program did you use to convert it to Mobi?

**Comment by kevzettler** [comment]
@skyace65 i'm on an old version of Calibre 6.20 in order to update i'd have to update my operating system (MacOS 12.4) and I'm hesitant because that may break all my current work flows. I'd expect a table of contents wouldn't need bleeding edge Calibre support.

Anyways. I was able to find the table of contents in the Calibre viewer. 
![image](https://github.com/godotengine/godot-docs/assets/126209/13c943cf-7326-4b19-a6fd-0b6466eb2369)
Had to go looking for it in the viewer features

I was able to get it rendering on Kindle. I had to convert the Epub to AZW3 and during the conversion there's a setting to like "move generated table of contents to the front instead of end" that I selected. Not sure if that was needed I didn't compare default MOBI vs AZW3 output. Maybe this comment will be helpful to others ( or myself in the future).
